docs/UserGuide.md

```diff
@@ -473,20 +497,26 @@ Examples:
 
 Performs operations on contacts in the address book that match the given condition.
 
-Format: `batch COMMAND where/CONDITION`
+Format: `batch COMMAND by/FIELD [=/EQUALS_VALUE] [start/START_WITH] [end/END_WITH]`
 
-* The allowed operations in `COMMAND` are
-  * list
+* `COMMAND` must be a valid command without `INDEX`. The allowed operations in `COMMAND` are:
   * edit
   * delete
-* The `CONDITION` field must conform to the following syntax: `TERM OP TERM`
-  * Valid operators for the `OP` field are `>`, `<`, `=`, `!=`, `LIKE`
-  * A `TERM` may be an attribute of a person or a constant value
+* The `FIELD` provided must be provided, and must match one of the following:
+  * `Name`
+  * `Address`
+  * `Phone`
+  * `Email`
+* A `EQUALS_VALUE` is value matching the exact value in the target field.
+  * Only full words will be matched e.g. `Han` will not match `Hans`.
+  * If this value is provided, `START_WITH` and `END_WITH` will be ignored.
+* A `START_WITH` is value matching the value in the target field with specific starting value.
+* A `END_WITH` is value matching the value in the target field with specific ending value.
```

Comment:
Are they mutually exclusive? (i.e. you can only specify one).

Reply:
Also, can you specify 2 different parameters in the same command?

Reply:
the if else check only followed equals if equals provided, but no strictly only one must be provide can change to mutually exclusive

Reply:
how can express the only one para can provide?